website/docs/faq.md

```diff
@@ -41,16 +58,67 @@ Go to the [Plans](https://app.configcat.com/organization/plans) page and use the
 You need to have a [Billing Manager](organization.md#billing-manager-role) role to access the Plans page.
 
 ## Security
-### I can't login to ConfigCat, because I was asked a 2FA (Two-factor authentication) code that I don't know.
+### Are you ISO certified? 
+We can proudly say that as of 2022 May 31, ConfigCat has achieved the ISO/IEC 27001:2013 certification for Information Security Management Systems (ISMS).
 
-*Solution 1:* There might be an authenticator app on your phone that you can use to login to ConfigCat.
+### I'm configuring my firewall, which addresses should I whitelist?
```

Comment:
This list might be outdated, ask Lali please.
Or reference a page with the up to date URLs.